game-core/src/test/java/games/strategy/engine/message/unifiedmessenger/RemoteMessengerTest.java

```diff
@@ -145,7 +146,7 @@ public void testRemoteCall() throws Exception {
       final TestRemote testRemote = new TestRemote();
       serverRemoteMessenger.registerRemote(testRemote, test);
       // since the registration must go over a socket and through a couple threads, wait for the client to get it
-      await().until(() -> unifiedMessengerHub.hasImplementors(test.getName()));
+      await().until(() -> unifiedMessengerHub.hasImplementors(test.getName()), is(true));
```

Comment:
The changes in this file weren't absolutely necessary, but they do make the failure messages slightly more readable.